materials/module-1/presentation/part-1/mesh-topology.md

```diff
@@ -0,0 +1,22 @@
+# Mesh topology
+
+```
+                                                +---------+
+                                                | Dropbox |
+               Last-mile mesh network           | data    |
+                                                | centre  |
+                                  +----------+  +---------+
+                     +------+     | Other    |    ^  ^
+                     |      |---->| internet |    |  |
+       +------------>| +------+   | service  +----+  | Internet
+       |             +-|      |-->| provider |       |
++------+--+----------->| +------+ +----------+  +----+-----+
+| Your    |            +-| Mesh |               | Internet |
+| home    +------------->| peer +-------------->| service  |
+| devices |              +------+               | provider |
++---------+  https://dropbox.com/some-resource  +----------+
+```
+
+- [Great Firewall of China](https://en.wikipedia.org/wiki/Great_Firewall) routinely scans, censors, and modifies internet traffic
```

Comment:
These could be streamlined (and maybe a slide with an image and you saying these words would be more evocative?)